BUILD.md

```diff
@@ -161,9 +161,7 @@ npm run start:api-server
 ```
 
 After starting the API server, you will see in the logs that plugins were loaded
-and that the API is reachable on the port you specified (4000 by default) and
-the Web UI (Cockpit) is reachable through port on the port your config
-specified (3000 by default).
+and that the API is reachable on the port you specified (4000 by default).
```

Comment:
@AzaharaC It would be worth documenting the functionality still and just mention that by default it is off and that it can be turned on with a specific  flag.